GSoC-2025/brainglobe: Update atlas packaging scripts to atlas API v2 (Swapnaneel Patra).md

```diff
@@ -0,0 +1,99 @@
+
+## brainglobe: Update atlas packaging scripts to atlas API v2 (Swapnaneel Patra)
+
+## Personal details
+
+- **Full name**: Swapnaneel Patra
+- **Email**: <swapnaneel06@gmail.com>
+- **GitHub username**: thisisrick25
+- **Zulip username**: Swapnaneel
+- **Location & time-zone**: Kolkata, India, GMT+5:30
+- **Personal website / project portfolio**: [github.com](https://github.com/thisisrick25)
+- **Code contribution**: [https://github.com/brainglobe/brainglobe-atlasapi/pull/555](https://github.com/brainglobe/brainglobe-atlasapi/pull/555)
+- **Proposal discussion link**: https://github.com/neuroinformatics-unit/gsoc/pull/56
+
+## Project proposal
+
+_Length: max 1 page_
+
+- **Synopsis**
+
+This project aims to update the BrainGlobe Atlas API by integrating the OpenMINDS SANDS standard for neuroanatomical atlases.
+Currently, atlas packaging scripts convert public atlas data to a limited standard format; by adapting these scripts to produce output compliant with OpenMINDS SANDS,
+The project will enable consistent, interoperable data representations across BrainGlobe tools.
```

Comment:
We already have this. Adopting the new standard is more about integrating with the wider software ecosystem